md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/LineageAPI.java

```diff
@@ -215,7 +210,7 @@ RestWrapper getByInstanceExec(@RequestParam(value = "ied", defaultValue = "0") S
 
             SimpleDateFormat format = new SimpleDateFormat("MM-dd-yyyy HH:mm:ss");
 
-            StringBuffer InstanceExecLabel = new StringBuffer("<<TABLE ALIGN=\"LEFT\" CELLSPACING=\"4\" CELLPADDING=\"0\" BORDER=\"0\" WIDTH=\"100%\">" +
+            StringBuilder InstanceExecLabel = new StringBuilder("<<TABLE ALIGN=\"LEFT\" CELLSPACING=\"4\" CELLPADDING=\"0\" BORDER=\"0\" WIDTH=\"100%\">" +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)